app/api/authentication.py

```diff
@@ -70,3 +73,45 @@ def auth_status():
             'user_id': getattr(current_user, 'user_id', None),
         },
     }), constants.api.SUCCESS_CODE
+
+
+def oidc_request_loader():
```

Comment:
This function is not an API handler and doesn't belong in this file. Maybe create a new file `app/util/oidc.py` and move this there?